cicada/commands/exec_schedule.py

```diff
@@ -224,6 +224,19 @@ def main(schedule_id, dbname=None):
                             minutes=db_conn_alert_delay
                         )
 
+            if returncode != 0:
```

Comment:
Will this exclude cases where cicada issues a SIGTERM to the sub-process ?

Reply:
cicada SIGTERM cases return code -15 and it won't be excluded in this step. but user can exclude -15 by `return code_alert` in yaml file.
We can exclude it completly but in that case won't have the option to send alert if a test is manually aborted. 
which one do you think is better? option to exclude it in yaml file as it is, or hard coded excluding ?